### ISSUE TITLE
Flicking and tapping opens menu 

### ISSUE THREAD

**Issue by JakeDevs** [tachiyomiorg/tachiyomi#9506]
### Steps to reproduce

Flick and tap while reading manga

### Expected behavior

The menu should not open 

### Actual behavior

The menu opens 

### Tachiyomi version

0.14.6

### Android version

Android 13

### Device

Samsung s21

### Other details

I'm not sure if this is an issue with tachiyomi or device specific, however I think a lock option should be implemented if it is device specific.

In the example below I'm using aniyomi but it still occurs on tachiyomi 0.14.6 and the latest version of tachiyomi sy.

https://github.com/tachiyomiorg/tachiyomi/assets/91037748/12c40195-95cb-4507-bbb2-cf78e3c5a83b

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by JakeDevs** [comment]
On further testing, this issue seems to be exclusive to tachiyomi and apps based on it. Other manga readers work well.

**Comment by arkon** [comment]
I can't quite tell from the recording, but can't you just not tap again if you don't want the menu to open, which happens if you tap?

**Comment by JakeDevs** [comment]
I don't think you understand. The flick is to make the screen quickly move in the desired direction, and the secondary tap is to cancel all movement to stop at a precise desired spot. What's going on here is not normal behavior.
